components/raftstore/src/store/config.rs

```diff
@@ -718,5 +728,11 @@ mod tests {
         cfg = Config::new();
         cfg.future_poll_size = 0;
         assert!(cfg.validate().is_err());
+
+        cfg = Config::new();
+        cfg.raft_base_tick_interval = ReadableDuration::secs(1);
+        cfg.raft_election_timeout_ticks = 9;
+        cfg.raft_store_max_leader_lease = ReadableDuration::secs(11);
```

Comment:
The test case will fail at L327 instead of L336.

Reply:
I have fix this, PTAL